nodeJS/express/installing_postgresql.md

```diff
@@ -0,0 +1,295 @@
+<!-- markdownlint-disable MD024 TOP004 -->
+
+### Introduction
+
+Data persistence is integral to the vast majority of web applications. Data persistence is achieved through databases. Being able to understand how to structure, build, and query your own database are important skills for any full-stack developer to have.
+
+For example, consider The Odin Project. Where is your username stored? Where are project submissions stored? Where does this lesson content, the one you're reading right now, stored? Yes, a database!
+
+We've chosen [PostgreSQL](https://www.postgresql.org/) as our database of choice for the curriculum. All SQL knowledge that you've learned in the earlier course should be transferable to PostgreSQL. We'll also learn new PostgreSQl specific things. Watch through this [fun and short introduction video to PostgreSQL by Fireship](https://www.youtube.com/watch?v=n2Fluyr3lbc).
+
+<div class="lesson-note lesson-note--critical" markdown="1">
+
+#### Prerequisite SQL knowledge
+
+Make sure you've completed the [SQL course](https://www.theodinproject.com/paths/full-stack-javascript/courses/databases).
+
+This lesson and all subsequent lessons will assume you understand SQL syntax and concepts.
+
+</div>
+
+In this lesson, we'll install PostgreSQl and in the next lesson we'll learn how to use it in our Express application.
+
+### Lesson overview
+
+This section contains a general overview of topics that you will learn in this lesson.
+
+- How to install PostgreSQL.
+
+### Installing PostgreSQL
+
+If you've been following the curriculum, you should already have installed all the supporting bits and pieces. If not, please go back and refer to [those lessons](https://www.theodinproject.com/guides/installations) before continuing.
```

Comment:
Do we need this paragraph? It's in the Rails version of this lesson but I've got a [note in my descriptive links spreadsheet for the Rails pathway](https://docs.google.com/spreadsheets/d/1rM9tjPIFZbKoMfd6OlLnjOhddBe2bivllY8_Bg5aGqY/edit#gid=1290166870&range=F30) that questions its value, given this curriculum is supposed to be taken linearly. I guess this probably also applies to things like the prerequisite boxes in both lessons.

At the very least, if we decide this paragraph is actually necessary, we'd need to change the link text to be more descriptive than "those lessons" (which will of course be changed accordingly in the Rails' equivalent when we get to that epic).

Reply:
agreed. removed the paragraph.